network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -54,6 +55,7 @@ public NetworkStoreRepository(DataSource dataSource, ObjectMapper mapper, Mappin
         this.dataSource = dataSource;
         this.mapper = mapper;
         this.mappings = mappings;
+        mapper.registerModule(new JodaModule());
```

Comment:
shouldn't be in the constructor used by spring because spring already registers the joda bean you declared in networkstoreapplication I think